data_safe_haven/resources/workspace/workspace.cloud_init.mustache.yaml

```diff
@@ -93,6 +93,16 @@ write_files:
       ansible-playbook /desired_state/desired_state.yaml
       popd
 
+  - path: "/root/gitea.url"
+    permissions: "0400"
+    content: |
+      URL=http://{{gitea_hostname}}
+
+  - path: "/root/hedgedoc.url"
+    permissions: "0400"
+    content: |
+      URL=http://{{hedgedoc_hostname}}
+
```

Comment:
Let's do this a more sensible way

Reply:
I think we need a general tidy-up of how we pass runtime variables to ansible, but I'm not sure this PR is the right place to put it. I think this is captured in #2075 - do you agree?

Reply:
I think I'd like to try it on this PR (I ran out of time yesterday). If it works then follow that pattern elsewhere, if not, merge this as is.

Reply:
Are you still looking into this? I see you made some commits - did they work?

Reply:
Yep, testing this now.